frontend/ios/Runner/Info.plist

```diff
@@ -28,14 +28,12 @@
 	<false/>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
-	<key>NSCameraUsageDescription</key>
```

Comment:
move to language specific. I think we don't use anything from photo library @michael-markl ? Not sure why this string exists

Reply:
A few years ago ( :D ) we had some application form implemented in the native app. So, that's probably a left over of that.